evergreen/config.yml

```diff
@@ -1300,6 +1325,7 @@ buildvariants:
     run_with_encryption: On
     enable_tsan: On
     cmake_build_type: RelWithDebInfo
+    disable_tests_against_baas: On
```

Comment:
Why? Is it just very slow?

Reply:
Hmm. I think this is a mistake. It's harmless since we've never run the baas tests here anyways. It probably speeds compiles up a bit.